src/components/Results.vue

```diff
@@ -121,6 +121,11 @@ export default {
         ManipulationCategory.fromName(categoryName),
       );
     },
+    beforeRouteLeave(to, from, next) {
```

Comment:
Dużo lepiej 👍
Natomiast `beforeRouteLeave` żeby zadziałał nie powinien być w `methods`, tylko zdefiniowany w top level obiektu komponentu. Przykład tutaj: https://router.vuejs.org/guide/advanced/navigation-guards.html#in-component-guards

Reply:
o, coś się wkradło jak jest gdzie indziej

Reply:
No jak testowałem na Twoim branchu i przeniosłem tę funkcję poziom wyżej to wszystko działało gites. Mogę dorzucić commita jeśli trzeba.